meetings/2024-04/README.md

```diff
@@ -0,0 +1,26 @@
+# 101st meeting of Ecma TC39 Summary
+
+## Proposals
+
+| Advanced to     | Proposal                                      |
+|-----------------|-----------------------------------------------|
+| Stage 4         | Set methods                                   |
+| Stage 4         | duplicate named capture groups                |
+| Stage 3         | new Function                                  |
+| Stage ~~2.7~~ 3 | Make eval-introduced global vars redeclarable |
```

Comment:
```suggestion
| Stage 3 | Make eval-introduced global vars redeclarable |
```

I was confused by this and had to review the notes to understand what this meant 🙂

Reply:
Fair enough. I wanted to record that it advanced twice in the meeting with minimal confusion, which… I clearly failed. 😆